core/src/page_id.rs

```diff
@@ -119,6 +119,13 @@ impl PageId {
         Ok(PageId { path })
     }
 
+    /// Get the child index of the page at the given depth.
+    ///
+    /// This panics if the depth of the page is not at least `depth + 1`.
+    pub fn child_index_at_level(&self, depth: usize) -> u8 {
```

Comment:
This looks as if it should return `ChildPageIndex`. If there is a reason, please document it here, along with the promise that the returned value is ≤ MAX_CHILD_INDEX.